OMEdit/OMEditLIB/OMS/OMSSimulationOutputWidget.cpp

```diff
@@ -129,7 +129,7 @@ OMSSimulationOutputWidget::OMSSimulationOutputWidget(const QString &cref, const
     OMSimulatorPage *pOMSimulatorPage = OptionsDialog::instance()->getOMSimulatorPage();
     int logLevel = pOMSimulatorPage->getLoggingLevelComboBox()->itemData(pOMSimulatorPage->getLoggingLevelComboBox()->currentIndex()).toInt();
     args << QString("--logLevel=%1").arg(logLevel);
-    args << QString("--option=%1").arg(pOMSimulatorPage->getCommandLineOptionsTextBox()->text());
+    args << QString("--option \"%1\"").arg(pOMSimulatorPage->getCommandLineOptionsTextBox()->text().replace(" ", "\" \""));
```

Comment:
So `=` is not needed, right?

Reply:
Right, it is not needed. I intentionally kept it for the other arguments though, because they just get a single value assigned.